AVS-Landing-Zone/GreenField/ARM/ESLZDeploy.deploy.json

```diff
@@ -53,7 +53,7 @@
       "type": "array",
       "defaultValue": [],
       "metadata": {
-        "description": "Email addresses to be added to the alerting action group"
+        "description": "Email addresses to be added to the alerting action group. Use the format [\"name1@domain.com\",\"name2@domain.com\"]."
```

Comment:
Can we also update the bicep file (https://github.com/Azure/Enterprise-Scale-for-AVS/blob/main/AVS-Landing-Zone/GreenField/Bicep/ESLZDeploy.bicep#L20) and then run the [Build-ARM script](https://github.com/Azure/Enterprise-Scale-for-AVS/blob/main/Build-ARM.ps1) in the root of the repo, this will update the ARM template content hash values.

Reply:
thanks @ScottHolden. i have updated the bicep file as well as ran the build-arm script.

Reply:
Thanks, can I get you to try running `az bicep upgrade`, and then running the build script again? Looks like you have an older version of Bicep (0.3.255, latest should be either 0.4.1124 or 0.4.1008, you can check using `az bicep version`).

I'll raise an issue to see if we can auto-build the templates upon pull request to make things easier going forward (or at least warn of the version in the script) 🙂